nodejs/eks/nodegroup.ts

```diff
@@ -1815,6 +1843,19 @@ function createManagedNodeGroupInternal(
         );
     }
 
+    const amiIdMutuallyExclusive: (keyof Omit<ManagedNodeGroupOptions, "cluster">)[] = [
+        "gpu",
+        "amiType",
+    ];
+    amiIdMutuallyExclusive.forEach((key) => {
+        if (args.amiId && args[key]) {
+            throw new pulumi.ResourceError(
+                `amiId and ${key} are mutually exclusive to create a managed node group`,
```

Comment:
The grammar here seems a little funny (but also I'm not a native speaker). Feed it to LLM?

Reply:
I actually copied the grammar from a similar error message from above. But now that you're saying this it also sounds weird to me (also not native speaker).

Let me reword those.